.travis.yml

```diff
@@ -1,3 +1,9 @@
+branches:
+  only:
+    - master
+    - /^release-\d+\..+$/
+    - /^v\d+\..+$/
```

Comment:
What branch is signified with `^v...` here? The branches I see in our repo only have `master` or `release-*`.

Reply:
I believe this is to match tags - Travis will fire on a new tag.

Reply:
yaml and comments :disappointed: 

According to the documentation, if I add a safelist (branch -> only) and I tag a commit on one of those branches, the process won't trigger (because it will only look at the tag and not the branch name?) so it's necessary to a pattern for the tags that would be applied to the corresponding branches. This obviously has the side effect of triggering the process for tags on any branch...